content/_index.md

```diff
@@ -126,3 +126,16 @@ on the Kubernetes Slack, or join the [Dex-dev][dex-dev] mailing list.
 [dex-dev]: https://groups.google.com/forum/#!forum/dex-dev
 [slack]: slack://channel?team=T09NY5SBT&id=C011URMR41W
 [disclosure]: https://coreos.com/security/disclosure/
+
+---
+We are a Cloud Native Computing Foundation sandbox project.
+
+{{< rawhtml >}}
```

Comment:
I couldn't find a reasonable way to have an image resized in markdown and contain a link -- I tried `[![CNCF](img/logos/cncf-color.png =200x)](https://cncf.io)`, but that didn't work. I'm also not sure of any better way to use raw HTML in the markdown. I'd be happy for an alternative to this.

Reply:
It would be better to implement this as a partial (`partials/cncf-status.html`) and add it to the (by creating a) homepage template (`layouts/_default/index.html`.) 

This is actually a bit of a complicated operation, so it might be best if I give it a try?

Reply:
I’d be happy for you to give it a try :)

I was looking at a way to add a [cncf.status] in the confit.toml, but couldn’t see a way to access it in the markdown. Initially I had tried to have the CNCF stuff in the footer, but it doesn’t need to be on every page like the copyright stuff does.